internal/jimm/jimm.go

```diff
@@ -41,16 +42,13 @@ var (
 	}
 )
 
-// A JIMM provides the business logic for managing resources in the JAAS
-// system. A single JIMM instance is shared by all concurrent API
-// connections therefore the JIMM object itself does not contain any per-
-// request state.
-type JIMM struct {
+// Parameters holds parameters for the JIMM type.
```

Comment:
Let's better explain this. 
```suggestion
// Parameters holds the services and static fields passed to the jimm.New() constructor.
// You can provide mock implementations of certain services where necessary for dependency injection.
```